src/themes/classic/components/button.js

```diff
@@ -7,7 +7,7 @@ import { defaultControlStyles } from '../deprecated/shared'
 const baseStyle = {
   fontFamily: 'fontFamilies.ui',
   borderRadius: 'radii.1',
-  border: 0,
+  border: '0',
```

Comment:
Hm... Actually yeah this is still invalid. Could be 'none'

Reply:
Same effect as `border: 'none'` though? One sets border style, the other sets border width (effectively) - point is to show _no_ border, and `Box`s prop types don't know how to handle `border={number}`